client/my-sites/email/email-non-domain-owner-message/style.scss

```diff
@@ -1,4 +1,9 @@
-.email-non-domain-owner-message__non-owner-message {
+.email-non-domain-owner-message__promo-card {
+	margin-top: 16px;
+	padding-bottom: 0;
+}
+
+.email-non-domain-owner-message__text {
```

Comment:
Can't we just merge those two classNames? They seem to be equivalent.

What about call it `email-non-domain-owner-message__style`

Reply:
They're very close, but there's a difference in `padding-bottom` vs `margin-bottom`.

Reply:
Right, subtle. But from my test I saw when merging it into one (don't remember which one) the visual effect was the same.

Reply:
There is actually a visual difference. Here's what the promo card would look like with those styles:

![promo card](https://user-images.githubusercontent.com/1101677/198267391-bba408cc-85d0-4148-85d0-63d58efeeddc.png)